ButtonView.h

```diff
@@ -0,0 +1,44 @@
+#import <Cocoa/Cocoa.h>
+
+@protocol ButtonActionHandler <NSObject>
+- (void)generateRandomRectangleForWindow:(NSWindow *)window;
+@end
+
+@interface ButtonView : NSButton
+
+/* MARK: - Properties */
```

Comment:
What are these unused properties? Why did you declare them and not implement them?

Reply:
Those are for future implementations...